packages/server-web/src/client/component/TodoItem.vue

```diff
@@ -115,7 +115,7 @@ export default {
   },
   methods: {
     cancel() {
-      this.updating = false;
+      this.$store.commit("isEditing", { [this.$props.todo.todoId]: false });
```

Comment:
Do we still need `updating: false` in `data()` ?

Reply:
good catch